dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -846,6 +849,47 @@ public void testManualRegisterThenPublish() throws ApiException {
         workflowApi.publish(bitbucketWorkflow.getId(), publishRequest);
     }
 
+    //TODO: Fork these workflows onto dockstoretestuser
```

Comment:
Do this before a merge.
I would actually suggest https://github.com/dockstore-testing (in this case, I might be wrong but I don't think the test users need to own these workflows)

Reply:
forked